<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLine</class>
 <widget class="QDialog" name="AddLine">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/icon/7.PNG</normaloff>:/icon/icon/7.PNG</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab_1">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <attribute name="title">
     <string>线路</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入线路名称：</string>
     </property>
     <property name="buddy">
      <cstring>lineEditLineName</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLineName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加线路</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择线路颜色：</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonChooseColor</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonChooseColor">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择线路颜色</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>站点</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxLatitude">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>30.899999999999999</double>
     </property>
     <property name="maximum">
      <double>31.449999999999999</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>30.899999999999999</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelLatitude">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>纬度：</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLatitude</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="labelLongitude">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>经度：</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLongitude</cstring>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxLongitude">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>121.000000000000000</double>
     </property>
     <property name="maximum">
      <double>122.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>121.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditStationName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>站点名称：</string>
     </property>
     <property name="buddy">
      <cstring>lineEditStationName</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加站点</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择添加到线路：</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>连接</string>
    </attribute>
    <widget class="QComboBox" name="comboBoxConnectStation1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>站点1：</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectStation1</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>站点2：</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectStation2</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConnectStation2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>连接两个站点</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>选择连接线路：</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectLine</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConnectLine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>文本方式</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
